teamcity: failed test: TestLogic

The following tests appear to have failed on master (test): TestLogic, TestLogic/fakedist: TestLogic/fakedist/schema_change_in_txn, TestLogic/fakedist: TestLogic/fakedist/schema_change_in_txn: TestLogic/fakedist/schema_change_in_txn/rollback_mutations, TestLogic/fakedist

You may want to check [for open issues](https://github.com/cockroachdb/cockroach/issues?q=is%3Aissue+is%3Aopen+TestLogic).

[#1137341](https://teamcity.cockroachdb.com/viewLog.html?buildId=1137341):


```
TestLogic
--- FAIL: test/TestLogic (402.170s)

------- Stdout: -------
    test_log_scope.go:81: test logs captured to: /tmp/logTestLogic210675704
    test_log_scope.go:62: use -show-logs to present logs inline



TestLogic/fakedist
--- FAIL: test/TestLogic: TestLogic/fakedist (0.620s)




TestLogic/fakedist: TestLogic/fakedist/schema_change_in_txn
--- FAIL: test/TestLogic: TestLogic/fakedist: TestLogic/fakedist/schema_change_in_txn (7.700s)

------- Stdout: -------
=== PAUSE TestLogic/fakedist/schema_change_in_txn



TestLogic/fakedist: TestLogic/fakedist/schema_change_in_txn: TestLogic/fakedist/schema_change_in_txn/rollback_mutations
--- FAIL: test/TestLogic: TestLogic/fakedist: TestLogic/fakedist/schema_change_in_txn: TestLogic/fakedist/schema_change_in_txn/rollback_mutations (2.010s)

------- Stdout: -------
--- done: testdata/logic_test/system with config fakedist: 56 tests, 0 failures
                logic.go:2311: 
                     
                    testdata/logic_test/schema_change_in_txn:711: 
                    expected "violates unique constraint \"i_idx\"", but found "pq: 1 entries, expected 3 violates unique constraint \"n_idx\""
                logic.go:2311: 
                     testdata/logic_test/schema_change_in_txn:714: too many errors encountered, skipping the rest of the input




```

Please assign, take a look and update the issue accordingly.

--- a/pkg/sql/logictest/testdata/logic_test/schema_change_in_txn
+++ b/pkg/sql/logictest/testdata/logic_test/schema_change_in_txn
@@ -707,7 +707,7 @@ CREATE UNIQUE INDEX i_idx ON customers (i)
 statement ok
 CREATE UNIQUE INDEX n_idx ON customers (n)
 
-statement error violates unique constraint "i_idx"
+statement error violates unique constraint
 COMMIT
 
 query TTBTTTB

